packages/@uppy/vue/src/utils.js

```diff
@@ -0,0 +1,3 @@
+export const isVue2 = (...args) => {
```

Comment:
I think we should ignore the warning in this case: This file is intended to be a bag of semi-related functions, it just happens to only contain one function right now.

```suggestion
// eslint-disable-next-line import/prefer-default-export
export const isVue2 = (...args) => {
```

Reply:
👌 makes sense!